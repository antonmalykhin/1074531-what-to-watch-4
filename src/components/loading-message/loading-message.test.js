import React from 'react';
import renderer from 'react-test-renderer';
import LoadingMessage from './loading-message.jsx';

it(`LoadingMessage render`, () => {
  const tree = renderer.create(
      <LoadingMessage />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
