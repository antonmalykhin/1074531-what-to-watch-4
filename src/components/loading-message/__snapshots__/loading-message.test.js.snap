// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingMessage render 1`] = `
<p
  style={
    Object {
      "color": "#eee5b5",
      "fontWeight": "bold",
      "textAlign": "center",
    }
  }
>
  Loading...
</p>
`;
